openslides/assignment/views.py

```diff
@@ -356,31 +379,31 @@ def append_to_pdf(self, story):
                 # List of assignments
                 for assignment in assignments:
                     story.append(Paragraph(
-                        escape(assignment.name), stylesheet['Heading3']))
+                        escape(assignment.title), stylesheet['Heading3']))
                 # Assignment details (each assignment on single page)
                 for assignment in assignments:
                     story.append(PageBreak())
                     # append the assignment to the story-object
                     self.get_assignment(assignment, story)
         else:  # print selected assignment
-            assignment = Assignment.objects.get(id=assignment_id)
+            assignment = Assignment.objects.get(pk=assignment_pk)
             # append the assignment to the story-object
             self.get_assignment(assignment, story)
 
     def get_assignment(self, assignment, story):
         # title
         story.append(Paragraph(
-            _("Election: %s") % escape(assignment.name), stylesheet['Heading1']))
+            _("Election: %s") % escape(assignment.title), stylesheet['Heading1']))
         story.append(Spacer(0, 0.5 * cm))
 
         # Filling table rows...
         data = []
-        polls = assignment.poll_set.filter(published=True)
+        polls = assignment.polls.filter(published=True)
         # 1. posts
         data.append([
             Paragraph("%s:" %
                       _("Number of available posts"), stylesheet['Bold']),
```

Comment:
Here the string has to be changed. Use the same string as in verbose_name in the model field.